rhizome/postgres/bin/initialize-database-from-backup

```diff
@@ -15,6 +15,9 @@ r "chown postgres /dat"
 r "rm -rf /dat/16"
 r "rm -rf /etc/postgresql/16"
 
+# Required to read server certificate and key
+r "usermod --append --groups cert_readers postgres"
```

Comment:
Where do we add `prometheus` user to `cert_readers` group?

Reply:
We add it during the [image generation](https://github.com/ubicloud/postgres-images/pull/3/files#diff-3a42b2ee5f81a94b1675a1297a2532d118cbcf129eab64618a1d58a2b7fffa1fR423).

Reply:
On second thought, I'm removing this from here and add this to image generation as well.